### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by marcelaraujo** [dveselov/pyodconverter#3]
I'd just wanna say thank you for everything that you did to keep project in progress.

Thank so much!

**Comment by tdesvenain** [comment]
Thank you all ! Actually this project saves me as I use it in a new plone add-on collective.documentfusion
that generates pdf from odt files filling them with data. A long-time waited usecase.

**Comment by marcelaraujo** [comment]
I don't have enough acknowledge to do everything you do in Python but I did my best.
As soon as I can, I will help you again. If you don't matter, I wanna ask you to make a PR to my fork from yours to keep my fork update to.

Thank you again

PS: We use it to do some files previews and conversions on our system. This save my life and make a lot of free time for the team.
When I began look this project, it was hard but I learned so much Python.
